index.html

```diff
@@ -9,44 +9,74 @@
     <script src="https://kit.fontawesome.com/a12691952c.js" crossorigin="anonymous"></script>
 </head>
 <body>
-    <header>
+    <header class="header1">
         <nav class="decoration1">
             <img class="artdeco" src="resources/ic_menu.svg" alt="Toolbar menu">
         </nav>
-    </header>
-
-    <section>
-        <p class="p2">José Fernández-Caminero</p>
 
-        <p class="p1">I love Full-Stack Engineering and I can code websites that can be both the most complex and the most efficient.</p>
-
-        <button class="button1">Start collaboration</button>
+        <nav class="desktopnav">
+            <ul>
+                <li>Hello</li>
+                <img class="stickimg" src="resources/Rectangle52.svg" alt="Underlining stick">
+                <li>Portfolio</li>
+                <li>About</li>
+                <li>Contact</li>
+            </ul>
+        </nav>
+    </header>
 
-        <div class="decoration2">
+    <section class="dflex87">
+        <div class="dottedimg1">
+        </div>
+        <div class="decoration3">
         </div>
     </section>
     
-    <p class="p2">Projects</p>
-    <section>
-        <div class="box1">
-            <img class="noOF" src="resources/Rectangle 21.png" alt="Picture of laptop and other objects">
+    <section class="desktopflex">
+        <div class="styleName">
+            <p class="p2">José Fernández-Caminero</p>
+            
+            <p class="p1">I love Full-Stack Engineering and I can code websites that can be both the most complex and the most efficient. I'm addicted to coding challenges, algos, and data structures, and having fun. You could say I'm a machine that turns coffee into code.</p>
+            
+            <button class="button1">Start collaboration</button>
```

Comment:
- [ ] [OPTIONAL] i would suggest that you add a `cursor: pointer;` style to all of your button to change the cursor type when users hover on the buttons, that way your buttons will be more elegant and user friendly

before             |  after
:-------------------------:|:-------------------------:
![G1](https://user-images.githubusercontent.com/89282221/186756877-f7657a5c-d2ee-4421-ac8d-b34d524495eb.gif)  |  ![G2](https://user-images.githubusercontent.com/89282221/186756884-438d4a24-ddd3-40bc-87aa-73ced259cc57.gif)

Reply:
Awesome advice! Done!